templates/system/page--campaign.html.twig

```diff
@@ -0,0 +1,26 @@
+{% extends 'page.html.twig' %}
+
+{% block header %}
+  {% include '@croydon/system/header.html.twig' with {'page': page, 'border': false} %}
+{% endblock %}
+
+{% block content_inner %}
+
+  {% if page.messages %}
+    <div class="col-12 clearfix">
+      {{ page.messages }}
+    </div>
+  {% endif %}
+
+  <main {{ content_attributes.addClass('content col-md-8 order-2 order-md-1 mt-sm-5') }}>
+    {{ page.content }}
+
+    <a href="#main-content" class="action-link focus-top">Back to top</a>
+  </main>
+
+  <div class="sidebar col col-md-4 order-1 order-md-2 mt-sm-5">
+    {% if page.sidebar_second %}
+      {{ page.sidebar_second }}
+    {% endif %}
+  </div>
+{% endblock %}
```

Comment:
I don't think this template is being used either. Once #67  pull request is merged you will be able to override page templates for campaign pages by naming them `page--localgov-campaigns-overview.html.twig` and `page--localgov-campaigns-page.html.twig`